jjb/replica/replica.yaml

```diff
@@ -0,0 +1,194 @@
+---
+- defaults:
```

Comment:
Have you tested this? Because of the way JJB flattens all the files, this seems like it might be a name collision with global-defaults.yaml.

Reply:
yes.. this is actually on purpose since these pipelines belong to satura's org and not dent's 
Satura is going to have separate defaults for their pipelines. I also tested that this does not affect DentOS defaults.